src/overlays/fbdemos/ovl_fbdemo_triforce/z_fbdemo_triforce.c

```diff
@@ -16,16 +16,9 @@ void TransitionTriforce_SetType(void* thisx, s32 type);
 void TransitionTriforce_SetColor(void* thisx, u32 color);
 s32 TransitionTriforce_IsDone(void* thisx);
 
-#if 0
-TransitionInit TransitionTriforce_InitVars = {
-    TransitionTriforce_Init,
-    TransitionTriforce_Destroy,
-    TransitionTriforce_Update,
-    TransitionTriforce_Draw,
-    TransitionTriforce_Start,
-    TransitionTriforce_SetType,
-    TransitionTriforce_SetColor,
-    NULL,
+const TransitionInit TransitionTriforce_InitVars = {
```

Comment:
weren't you going to remove the `const`s?

Reply:
Yes. somehow they snuck back in.